pep-0582.rst

```diff
@@ -97,42 +97,46 @@ works. For example, ``__pypackages__`` will be ignored if the ``-P`` option or
 the ``PYTHONSAFEPATH`` environment variable is set.
 
 In order to be recognised, the ``__pypackages__`` directory must be laid out
-according to a new ``localpackages`` scheme in the sysconfig module.
-Specifically, both of the ``purelib`` and ``platlib`` directories must be
-present, using the following code to determine the locations of those
-directories::
+according to a new ``local_packages`` scheme in the sysconfig module. The scheme
+will provide the ``purelib`` and ``platlib`` paths, but is not required to
+provide any additional paths.
 
-    scheme = "localpackages"
-    purelib = sysconfig.get_path("purelib", scheme, vars={"base": "__pypackages__", "platbase": "__pypackages__"})
-    platlib = sysconfig.get_path("platlib", scheme, vars={"base": "__pypackages__", "platbase": "__pypackages__"})
+The following code can be used to determine the locations of the directories::
 
-These two locations will be added to ``sys.path``, other directories or
-files in the ``__pypackages__`` directory will be silently ignored. The
-paths will be based on Python versions.
+    >>> paths = sysconfig.get_paths("local_packages", vars={"base": ".", "platbase": "."})
+    >>> paths["purelib"], paths["platlib"]
+    ("./__pypackages__/3.10/site-packages", "./__pypackages__/3.10/site-packages")
 
-.. note:: There is a possible option of having a separate new API, it is documented at `issue #3013 <https://github.com/python/peps/issues/3013>`_.
+
+An absolute version of these two locations will be added to ``sys.path``, other
+directories or files in the ``__pypackages__`` directory will be silently
+ignored. The paths will be based on Python versions.
+
+.. note::
+
+    There is a possible option of having a separate new API, to avoid
+    `sysconfig` having to provide other scheme paths. It is documented in
+    `Kushal's fork <https://github.com/kushaldas/peps/issues/3>`_.
 
 
 Example
 -------
 
 The following shows an example project directory structure, and different ways
 the Python executable and any script will behave. The example is for Unix-like
-systems - on Windows the subdirectories will be different.
+systems - on Windows the subdirectories can be different.
```

Comment:
```suggestion
systems --- on Windows the subdirectories can be different.
```

Minor typographical nit, but should be an em dash rather than a hyphen here